### ISSUE TITLE
update results not found copy on search

### ISSUE THREAD

**Issue by mikefantini** [DoSomethingArchive/legacy-website#3304]
assigning to me for now as I'm waiting on feedback from Ben

**Comment by mikefantini** [comment]
suggested copy here: https://docs.google.com/a/dosomething.org/document/d/1qzPPflSebgzjQih_c_IVPOptwqkCKG_xgwpJmdTwGK0/edit

**Comment by mikefantini** [comment]
@blisteringherb is this a cofig variable anywhere where I can just update the copy?

**Comment by mikefantini** [comment]
no longer needed since we'll be using the no results found config variable per #3497
